pkg/api/v1/types.go

```diff
@@ -988,6 +990,12 @@ type Container struct {
 	// Whether this container should allocate a TTY for itself, also requires 'stdin' to be true.
 	// Default is false.
 	TTY bool `json:"tty,omitempty"`
+	// Whether this container should send stdout.
```

Comment:
`stdout` is to tell if container out is sent to stdout. The attach ability is not used here because the container could just exit after exec, attaching to an exited container just fails.